Adição do componente Flutuar

gabrielrudan opened the pull request.

gabrielrudan committed:
Adicionando o script de Flutuar

italo-mendes commented on acoes/Flutuar.cs:
```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flutuar : MonoBehaviour
```
Remover os valores iniciais das variáveis, assim como foi solicitado e exemplificado nos componentes enviados como exemplo.

Além disso, acredito que o código pode funcionar da mesma forma para jogos 2D e 3D, sendo assim, pode-se alterar o código para 3D que se tornará mais abrangente.

italo-mendes commented on acoes/Flutuar.cs:
```diff
@@ -0,0 +1,37 @@
+    public float amplitude = 0.25f;
+
+    //Frequ�ncia de "Eleva��es" do objeto
+    public float frequencia = 1f;
```
alterar o nome da variável para **frequenciaElevacoes**, a fim de que a variavel por se só já seja auto explicativa sem necessitar do comentário, visto que o objetivo é que os componentes sejam usados sem visualizar o código

italo-mendes commented on acoes/Flutuar.cs:
```diff
@@ -0,0 +1,37 @@
+public class Flutuar : MonoBehaviour
+{
+    //Graus de rota��o por segundo
+    public float grausPorSegundo = 20.0f;
```
Observando o código, o gameObject não rotacionará esse valor em graus por segundo, sendo assim, é mais interessante alterar o nome da variável para **velocidadeRotacao**.

italo-mendes committed:
Update Flutuar.cs

Alterações para simplificar o código sem perda de eficiência e funcionalidade, pois o que pretende-se é rotacionar o gameObject e movimentá-lo no eixo Y de acordo com a função seno. A posição iniciar não era necessária para manter o movimento de sobe e desce no eixo Y.
